src/display_device.h

```diff
@@ -0,0 +1,87 @@
+/**
+ * @file src/display_device.h
+ * @brief Declarations for display device handling.
+ */
+#pragma once
+
+// local includes
+#include "platform/common.h"
+
+// forward declarations
+namespace display_device {
+  template <class T>
+  class RetryScheduler;
+  class SettingsManagerInterface;
+}  // namespace display_device
+
+namespace display_device {
+  /**
+   * @brief A singleton class for managing the display device configuration for the whole Sunshine session.
+   *
+   * This class is meant to be an entry point for applying the configuration and reverting it later
+   * from within the various places in the Sunshine's source code.
+   */
+  class session_t {
+  public:
+    /**
+     * @brief Get the singleton instance.
+     * @returns Singleton instance for the class.
+     *
+     * EXAMPLES:
+     * ```cpp
+     * session_t& session { session_t::get() };
+     * ```
+     */
```

Comment:
```suggestion
    /**
     * @brief Get the singleton instance.
     * @returns Singleton instance for the class.
     * @examples
     * session_t& session { session_t::get() };
     * @end_examples
     */
```

We have a new alias since you last contributed. See here: https://docs.lizardbyte.dev/projects/sunshine/en/master/source_code/source_code.html#guidelines

P.S. You can do this for libdisplaydevice if you want as well, otherwise I will eventually take care of it.